CMakeLists.txt

```
@@ -109,6 +109,7 @@ option(ENABLE_GEANT3_TESTING "Enable tests utilizing Geant3" OFF)
 option(BUILD_PROOF_SUPPORT "Support ROOT::Proof (deprecated)" OFF)
 
 option(BUILD_ONLINE "Build the online library" ON)
+option(BUILD_DATAMATCH "Build datamatch" OFF)
```

```suggestion
option(BUILD_DATAMATCH "Build datamatch" ON)
```

* Could you leave it at ON for the main commit?
* Then add another commit that explicitly disables it?
  That commit maybe should also add to the v19 ChangeLog?
* Then we can backport the first commit straight to 18.8

---

Done.